dotcms-integration/src/test/java/com/dotmarketing/portlets/templates/business/TemplateAPITest.java

```diff
@@ -1450,4 +1445,3822 @@ private HTMLPageAsset createNewVersion(final Variant variant, final HTMLPageAsse
         return APILocator.getHTMLPageAssetAPI()
                 .fromContentlet(checking);
     }
+
+    /**
+     * Method to test: {@link TemplateAPIImpl#saveAndUpdateLayout(Template, TemplateLayout, Host, User, boolean)}
+     * When: Save a Template with a null layout
```

Comment:
There's no null layout in the test

Reply:
done https://github.com/dotCMS/core/pull/28128/commits/5009888ac50550c0d87e2f4b5fbce1fdd7ba6c08#diff-06e01ead986cd9488744eacab14f2876283ace2f03afaedf556b306da79f094cR1448